INSTALLING.md

```diff
@@ -4,197 +4,271 @@ These instructions will guide you through the installation of Tenstorrent system
-> follow along with the documentation and tutorials to get started, you will
-> still need the source code.
->
-> If you do not want to use the models or follow the tutorials and want to
-> immediately start using the API, you may install just the wheel or get the release Docker container.
+---
 
-1. Install git and git-lfs.
+## TT-Metalium Installation
 
-```sh
-sudo apt install git git-lfs
-```
+#### There are three options for installing TT-Metalium:
+
+- [Option 1: From Source](#option-1-from-source)
+  Installing from source gets developers closer to the metal and the source code.
+
+- [Option 2: From Docker Release Image](#option-2-from-docker-release-image)
+  Installing from Docker Release Image is the quickest way to access our APIs and to start runnig AI models.
+
+- [Option 3: From Wheel](#option-3-from-wheel)
+  Install from wheel as an alternative method to get quick access to our APIs and to running AI models.
+
+--- 
 
-2. Clone the repo.
+## Option 1: From Source
+Install from source if you are a developer who wants to be close to the metal or the source code.
+
+### Step 1. Clone the Repository:
 
 ```sh
 git clone https://github.com/tenstorrent/tt-metal.git --recurse-submodules
 cd tt-metal
 git submodule foreach 'git lfs fetch --all && git lfs pull'
 ```
 
-3. Install either from source, or from our release wheel. Note that if you are
-going to try using the model demos, we highly recommend you install from
-source.
+### Step 2. Setup Environment Variables and Invoke our Build Scripts:
 
-#### Option 1: From source
+- Note: Some advanced build configurations like unity builds require `CMake 3.20`.
+  To upgrade Cmake run:
+    ```sh
+    pip3 install cmake --upgrade
+    hash -r
+    cmake --version
+    ```
 
-We use CMake for our build flows.
+- Run the appropriate command for the Tenstorrent card you have installed:
 
-Set up the environment variables and invoke our build scripts. Note that for
-source builds, you must set these environment variables every time.
+| Card             | Command                              |
+|------------------|--------------------------------------|
+| Grayskull        | ```export ARCH_NAME=grayskull```     |
+| Wormhole         | ```export ARCH_NAME=wormhole_b0```   |
+| Blackhole        | ```export ARCH_NAME=blackhole```     |
 
-```sh
-export ARCH_NAME=<ARCH_NAME>
-export TT_METAL_HOME=$(pwd)
-export PYTHONPATH=$(pwd)
-./build_metal.sh
+- Then run: 
+  ```
+  export TT_METAL_HOME=$(pwd)
+  export PYTHONPATH=$(pwd)
+  ./build_metal.sh
+  ```
 
-# If you would like an out-of-the-box virtual environment to use,
+- (recomended) For an out-of-the-box virtual environment to use, execute:
+```
 ./create_venv.sh
 source python_env/bin/activate
 ```
 
-where `ARCH_NAME` is one of `grayskull`, `wormhole_b0`, or `blackhole`,
-depending on your Tenstorrent card type.
-
-> [!NOTE]
->
-> Note about Python environments: You do not have to use `create_venv.sh`. If you
```

Comment:
Why is this gone?

Reply:
The reasoning was that if you are less familiar with Python and its various environment tools, you will run the recommended command, otherwise you will know you can build it in a custom environment and that you may run into compatibility issues (a user running into errors for doing something other than what is recommended seems out of our scope). 

The previous text had a few (non-technical) issues but if the reasoning is wrong let me know what information exactly you would like to see here and I’ll add to it.

Reply:
Sure, I'm fine with that stance.

Users creating environments with methods outside of our recommendations is out of scope.